content/fr/integrations/deployments/qovery.md

```diff
@@ -1,70 +1,70 @@
+[Qovery](https://qovery.com) est une plate-forme cloud entièrement gérée qui s'exécute sur votre compte AWS, GCP, Azure et Digital Ocean où vous pouvez héberger des sites statiques, des APIs backend, des bases de données, des tâches cron ou toutes autres applications en un seul endroit.
 
-Static sites are **completely free** on Qovery and include the following:
+Les sites statiques sont **entièrement gratuits** sur Qovery et incluent les éléments suivants :
 
-- Continuous, automatic builds & deploys from GitHub and GitLab.
-- Automatic SSL certificates through [Let's Encrypt](https://letsencrypt.org).
-- Free managed PostgreSQL.
-- Free SSD storage.
-- Unlimited collaborators.
-- Unlimited [custom domains](https://docs.qovery.com/guides/getting-started/setting-custom-domain/).
+- Constructions, déploiements continus et automatiques à partir de GitHub et GitLab.
+- Certificats SSL automatiques via [Let's Encrypt](https://letsencrypt.org).
+- PostgreSQL gratuitement géré.
+- Stockage SSD gratuit.
+- Collaborateurs illimités.
+- [Domaines personnalisés illimités](https://docs.qovery.com/guides/getting-started/setting-custom-domain/).
 
-## Prerequisites
+## Conditions préalables
 
-This guide assumes you already have a Nuxt project to deploy. If you need a project, follow the [Get Started](/docs/get-started/installation) guide.
+Ce guide suppose que vous avez déjà un projet Nuxt à déployer. Si vous avez besoin d'un projet, suivez le [guide de démarrage](/docs/get-started/installation).
 
-## Setup
+## Installation
 
-Follow the procedure below to set up Nuxt on Qovery:
+Suivez la procédure ci-dessous pour configurer Nuxt sur Qovery :
 
-### 1. Create a Qovery account.
+### 1. Créer un compote Qovery.
 
-Visit the [Qovery dashboard](https://console.qovery.com) to create an account if you don't already have one.
+Visitez le [dashboard Qovery](https://console.qovery.com) pour créer un compte si vous n'en avez pas déjà un.
 
-### 2. Create a project
+### 2. Créer un projet
 
-Click on "Create a new project" and give a name to your project.
+Cliquez sur "Créer un nouveau projet" et donnez un nom à votre projet.
 
-Click on "Next".
+Cliquez sur "Suivant".
 
-### 3. Add an application
+### 3. Ajouter une application
 
-Click on "Create an application" then choose "I have an application" and select the repository where your Nuxt site is located.
+Cliquez sur "Créer une application" puis choisissez "J'ai une application" et sélectionnez le référentiel où se trouve votre site Nuxt.
 
-Click on "Next".
+Cliquez sur "Suivant".
 
-Skip adding services for static website.
+Ignorez l'ajout de services pour le site Web statique.
 
-Click on "Deploy".
+Cliquez sur "Déployer".
 
-## Deploy
+## Déployer
 
 Your app should be deployed. You can see the status in real time by clicking on deployment logs.
```

Comment:
Could be translated: "Votre application devrait être déployée. Vous pouvez voir le statut en temps réel en cliquant sur les journaux de déploiement."

Reply:
I omit this, thanks a lot 🙏